### DISCUSSION TITLE
Não localizei o arquivo Blog.sql

### DISCUSSION THREAD

**Discussion by AndrePds** [balta-io/forum#64] [Tira Dúvidas] [answered]
Não localizai o arquivo Blog.sql usado no curso https://balta.io/player/assistir/8407be55-4137-48ae-baf5-470eef47a297/9480c859-13a7-4355-a736-d4843ca29b20, modeulo de autenticação e autorização. Poderia me informar onde eu encontro ?

Desde já agradeço!

**Comment by andrebaltieri** [comment, marked as answer]
Boa noite, @AndrePds, não seria este:
- https://github.com/balta-io/2811/blob/main/Modulo%203/Scripts/blog.sql
